### DISCUSSION TITLE
[Feature Request] Symbol for Folder's environments

### DISCUSSION THREAD

**Discussion by JA-nexDigital** [Kong/insomnia#3743] [Ideas]
Hello,

#### The story
While testing out Insomnia I encountered a small ~~problem~~ inconvenience. I created a few requests and added them to a folder, changed a environment variable for this folder specifically and forgot about them a few hours later. When I tried to come back to it and change some of the values it took me a minute to remember there was folder specific environment.

#### Proposal 
Would it be possible to add a small icon on the folder to notify the user that this folder uses a specific environment which is different from the Workspace selected environment ?

Best regards,

**Comment by gschier** [comment]
I agree that this would be very useful! Any ideas of how this could be signified in the UI?
